edxval/admin.py

```diff
@@ -69,9 +69,19 @@ def get_video_id(self, obj):
 
 
 class VideoTranscriptAdmin(admin.ModelAdmin):
-    list_display = ('video', 'language_code', 'provider', 'file_format')
+    raw_id_fields = ('video',)
+    list_display = ('get_video', 'language_code', 'provider', 'file_format')
+    search_fields = ('id', 'video__edx_video_id', 'language_code')
```

Comment:
Should we add `id` field to search ?

Reply:
its alright in case if someone just has an ID.